pkg/plugins/golang/v3/webhook.go

```diff
@@ -95,22 +102,37 @@ func (p *createWebhookSubcommand) InjectConfig(c config.Config) error {
 func (p *createWebhookSubcommand) InjectResource(res *resource.Resource) error {
 	p.resource = res
 
+	// by default, since the templates for the API have to be scaffolded, setting this to true.
+	p.doScaffold = true
+
 	p.options.UpdateResource(p.resource, p.config)
 
 	if err := p.resource.Validate(); err != nil {
 		return err
 	}
 
+	// verify conversion webhook configuration
+	if p.options.Spoke != "" {
+		if err := p.verifyConversionWHConfig(); err != nil {
+			return err
+		}
+		p.options.DoConversion = true
+	}
+
+	// Either defaulting, validating or conversion falgs need to be present, or the spoke version can be specified
 	if !p.resource.HasDefaultingWebhook() && !p.resource.HasValidationWebhook() && !p.resource.HasConversionWebhook() {
 		return fmt.Errorf("%s create webhook requires at least one of --defaulting,"+
-			" --programmatic-validation and --conversion to be true", p.commandName)
+			" --programmatic-validation, --conversion, and --spoke to be provided", p.commandName)
```

Comment:
Why and? Is not and/OR

Reply:
I'm not good at English so I just reflected [this comment](https://github.com/kubernetes-sigs/kubebuilder/pull/2084/files#r599412511). But as you said, I think `spoke` option is optional so like`fmt.Errorf("%s create webhook requires at least one of --defaulting,"+" --programmatic-validation and --conversion to be true. If using conversion webhook"+
				"you can specify the spoke versions in the command.", p.commandName)` is better?

Reply:
Is possible to use --conversion with spoke and --programmatic-validation with spoke or not?
So, I think we need to clarify it in the text.